nmmo/core/env.py

```diff
@@ -25,13 +26,16 @@ class Env(ParallelEnv):
 
   #pylint: disable=no-value-for-parameter
   def __init__(self,
-    config: Default = nmmo.config.Default(), seed=None):
+               config: Default = nmmo.config.Default(),
+               seed = None,
+               team_helper: TeamHelper = None):
```

Comment:
I don't like this approach, we're making env/realm depend on teams, but the only reason to do that is for spawning.

Could you just create baselines/lib/team/team_loader.py: 

class TeamLoader(SequentialLoader):
  __init__(team_helper)
  
 Then we can just modify nmmo_config.py to use that loader. That would make environment/realm agnostic to teams

Reply:
@daveey looking entity_manager.py, spawn code where loader is being used.

hmm, it's more tangled than I thought. Let me have a look.